azure-pipelines.yml

```diff
@@ -50,9 +50,6 @@ steps:
       npm run test
     displayName: "test"
 
-  - script: |
-      npm run lint
-    displayName: "lint"
```

Comment:
Why did you remove the linting?

Reply:
See above. The `npm run lint` command ran twice.